MobileFilter/sections/specific_web.txt

```diff
@@ -53,6 +53,7 @@ rakuraku-info.jp,sugoren.com,dailynewsonline.jp,yakuji.co.jp,asahi.com,m.winporn
 !
 wz.de##aside[data-portal-id="banner"]
 hentaiera.com##.bblocktop
+ascii.jp##.col-xs-4 > script[src^="//i.socdm.com/sdk/js/"]:upward(1)
```

Comment:
Better `#?#.post > script[src^="//i.socdm.com"]:upward(1)` and it should be in css_extended.

Reply:
ok,fixed